app/scripts/controllers/rif/lumino/index.js

```diff
@@ -52,43 +52,20 @@ export class LuminoManager extends AbstractManager {
         saveLuminoData: (data) => {
           luminoStorageHandler.saveLuminoData(data);
         },
-      };
+      }
       await this.lumino.init(signingHandler, storageHandler, configParams);
       const state = this.store.getState();
-      try {
         if (state.apiKey && !cleanApiKey) {
           await this.operations.setApiKey(state.apiKey);
-          try {
-            await this.operations.onboarding();
```

Comment:
It was wrong because of the explanation of Debray about how the sdk sets the api key, the sdk only needs the setApiKey call when you have already an api key, you don't need to run onboarding again so i removed all this, also this was causing problems for me so i leave it like the original version but without the onboarding call since is not necessary.